src/GitHub.App/ViewModels/GitHubPane/PullRequestCreationViewModel.cs

```diff
@@ -82,8 +82,11 @@ public class PullRequestCreationViewModel : PanePageViewModelBase, IPullRequestC
                 .Where(_ => TargetBranch != null)
                 .Subscribe(x =>
                 {
-                    if (!x.Any(t => t.Equals(TargetBranch)))
+                    // HACK: Why is `t` null?
+                    if (!x.Any(t => t != null && t.Equals(TargetBranch)))
```

Comment:
I'm not sure why `t` started coming up `null` when unit testing or whether isn't important. 😕

Reply:
I think this was caused by changing from an interface to a class. NSubstitute was automatically creating a substitute, so `t` wasn't `null`. I've fixed up the tests.